modules/xmpp/SDPUtil.js

```diff
@@ -547,6 +547,20 @@ const SDPUtil = {
     },
 
     /**
+     * Extracts the ICE username fragment from an SDP string.
+     * @param {string} sdp the SDP in raw text format
+     */
+    getUfrag(sdp) {
+        const ufragLines
+            = sdp.split('\n').filter(
+                    line => line.startsWith('a=ice-ufrag:'));
+
+        if (ufragLines.length > 0) {
+            return ufragLines[0].substr('a=ice-ufrag:'.length);
```

Comment:
given a line, there's already a function to parse the ufrag in sdputils: `parseICEUfrag`